int32 robotID
---
float32[2] potential

